apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -153,6 +153,14 @@ function ListBoxInline({ options, layout }) {
         setShowSearch(false);
       }
     };
+    if (isPopover) {
+      if (!selections.isActive()) {
+        selections.begin('/qListObjectDef');
+        selections.on('activated', show);
+        selections.on('deactivated', hide);
+      }
+      setShowToolbar(isPopover);
+    }
```

Comment:
maybe also add a check if selections exists. based on that the next if does maybe it is also needed here?